Terraform segment should work with opentofu

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

### What would you like to see changed?

The Terraform segment is awesome, but for those who use OpenTofu (https://opentofu.org/) it could use some tweaking: it looks for a file called "terraform", but for us it is called "tofu".

Until the new name is supported, we can make symlinks on both systems, aliases won't work.

Debug output on windows:
[ERROR] 10:21:58.580 terminal.go:CommandPath:321 → exec: "terraform": executable file not found in %PATH%
and on linux:
[ERROR] 10:27:36.109 terminal.go:CommandPath:321 → exec: "terraform": executable file not found in $PATH
as you would expect.

I'm writing this since I always create the symlinks but it's kind of annoying after a couple of times.
Thanks!

--- a/src/segments/terraform.go
+++ b/src/segments/terraform.go
@@ -32,7 +32,7 @@ type TerraformBlock struct {
 }
 
 func (tf *Terraform) Enabled() bool {
-	cmd := "terraform"
+	cmd := tf.props.GetString(Command, "terraform")
 	fetchVersion := tf.props.GetBool(properties.FetchVersion, false)
 
 	if !tf.env.HasCommand(cmd) || !tf.inContext(fetchVersion) {

--- a/website/docs/segments/cli/terraform.mdx
+++ b/website/docs/segments/cli/terraform.mdx
@@ -10,22 +10,25 @@ Display the currently active Terraform Workspace name.
 
 ## Sample Configuration
 
-import Config from '@site/src/components/Config.js';
+import Config from "@site/src/components/Config.js";
 
-<Config data={{
-  "type": "terraform",
-  "style": "powerline",
-  "powerline_symbol": "\uE0B0",
-  "foreground": "#000000",
-  "background": "#ebcc34",
-  "template": "{{.WorkspaceName}}"
-}}/>
+<Config
+  data={{
+    type: "terraform",
+    style: "powerline",
+    powerline_symbol: "\uE0B0",
+    foreground: "#000000",
+    background: "#ebcc34",
+    template: "{{.WorkspaceName}}",
+  }}
+/>
 
 ## Properties
 
-| Name            | Type      | Default | Description                                                                        |
-| --------------- | :-------: | :-----: | ---------------------------------------------------------------------------------- |
-| `fetch_version` | `boolean` | `false` | fetch the version information from `versions.tf`, `main.tf` or `terraform.tfstate` |
+| Name            |   Type    |   Default   | Description                                                                        |
+| --------------- | :-------: | :---------: | ---------------------------------------------------------------------------------- |
+| `fetch_version` | `boolean` |   `false`   | fetch the version information from `versions.tf`, `main.tf` or `terraform.tfstate` |
+| `command`       | `string`  | `terraform` | the command(s) to run, allows support for `tofu`                                   |
 
 ## Template ([info][templates])
 

